src/App.tsx

```diff
@@ -8,15 +8,17 @@ export const App = () => {
     <div className="App">
       <h1>Add todo form</h1>
 
-      <form action="/api/users" method="POST">
+      <form action="/api/todos" method="POST">
         <div className="field">
           <input type="text" data-cy="titleInput" />
           <span className="error">Please enter a title</span>
```

Comment:
The error message for the title input is always visible. Consider using conditional rendering to display this error message only when the input is invalid. This will improve user experience and accessibility.